### ISSUE TITLE
Problem install RPM: nothing provides libphp5

### ISSUE THREAD

**Issue by uasan** [nginx/unit#3]
Fedora 26
PHP 7.1 (installed: php-devel-7.1 php-embedded-7.1)

Try install package from repo
`dnf install unit`

Error:
```
Problem: conflicting requests
  - nothing provides libphp5-5.4.so()(64bit) needed by unit-0.1-1.el7.ngx.x86_64
```
How to fix it?

I guess i should try to compile everything with my hands :)

**Comment by banzayats** [comment]
I have the similar problem on my CentOS system (CentOS Linux release 7.3.1611).
PHP 5.6.31
```
# yum install unit                                                           
...                                              
Resolving Dependencies                                                                   
--> Running transaction check                                                            
---> Package unit.x86_64 0:0.1-1.el7.ngx will be installed                               
--> Processing Dependency: libphp5-5.4.so()(64bit) for package: unit-0.1-1.el7.ngx.x86_64
--> Finished Dependency Resolution                                                       
Error: Package: unit-0.1-1.el7.ngx.x86_64 (unit)                                         
           Requires: libphp5-5.4.so()(64bit)                                             
           Available: php-embedded-5.4.16-42.el7.x86_64 (base)                           
               libphp5-5.4.so()(64bit)                                                   
           Available: php-embedded-5.4.45-13.el7.remi.x86_64 (remi)                      
               libphp5-5.4.so()(64bit)                                                   
           Installed: php-embedded-5.6.31-1.el7.remi.x86_64 (@remi-php56)                
              ~libphp5-5.6.so()(64bit)                                                   
           Available: php-embedded-5.6.30-1.el7.remi.x86_64 (remi-php56)                 
              ~libphp5-5.6.so()(64bit)                                                   
```

**Comment by defanator** [comment]
@uasan currently we provide binary packages for CentOS 7 and Ubuntu 16.04 only. You can always build Unit from sources though.

@banzayats at these early days we are sticking to PHP, Python and Go versions available out of the box in each supported distro; in case of CentOS 7, the only supported PHP version is 5.4. You may want to disable remi's repository and remove `php-embedded` package installed from that repository, or try to build Unit from sources.

However, we are open to any reasoned suggestions here - e.g. we've been thinking about a way of distributing each Unit dynamic module in a separate package (as we already do for nginx dynamic modules). In such scenario one would be able to install Unit itself (the daemon) without any strong dependencies on additional packages provided by OS, configure 3rd-party repositories with newer versions of PHP/Python/whatever, and build their own module (or even module package for further redistribution).
